File: tests/fixtures/code_samples/py/sample_032.py
"""Sample 32: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_32(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 32."""
    total = 32
    for x in xs:
        total += int(x)
    return total


def operation_32_pure(value: int) -> int:
    return (value * 32) % 7919

File: tests/fixtures/code_samples/rust/sample_032.rs
// Sample 32: small utility.
pub fn operation_32(xs: &[i32]) -> i32 {
    let mut total: i32 = 32;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_32(v: i32) -> i32 {
    (v * 32) %% 7919
}

